src/js/components/Analytics.jsx

```diff
@@ -43,9 +45,17 @@ class Analytics extends Component {
   }
 
   componentWillMount () {
-    const {dispatch} = this.props
+    const {dispatch, user} = this.props
+    const event = {
+      group: 'pageView',
+      attributes: {
+        page: 'Analytics',
+        user: user.data.username
+      }
+    }
 
     dispatch(fetchSources())
+    dispatch(sendMetrics([event]))
```

Comment:
I am going to agree with @pml984 here. I don't want to have to modify each page to add the metrics. It should just happen when the routes change.